### PR TITLE
Support Defuse encryption

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -45,10 +45,15 @@ private function createAuthorizationServerNode(): NodeDefinition
                     ->defaultValue(null)
                 ->end()
                 ->scalarNode('encryption_key')
-                    ->info("The string used as an encryption key.\nHow to generate an encryption key: https://oauth2.thephpleague.com/installation/#string-password")
+                    ->info("The plain string or the ascii safe string used to create a \Defuse\Crypto\Key to be used as an encryption key.\nHow to generate an encryption key: https://oauth2.thephpleague.com/installation/#string-password")
```
Comment (X-Coder264) [inline DependencyInjection/Configuration.php]: You can use here sprintf() and Key::class

**Comment by X-Coder264** [comment]
@Nyholm Thank for the PR. Could you please add a bit of documentation for this so that people can see that this feature exists in the bundle? The reference configuration file in the `README.md` would also need to be updated to reflect the new state with the default values. Also it'd be good to add the `defuse/php-encryption` package in the composer.json `suggest` section.

**Comment by Nyholm** [comment]
Updated.

**Comment by Nyholm** [comment]
Thank you for merging.

**Pull request merged**